application/models/programme.php

```diff
@@ -802,7 +808,7 @@ public static function revision_diff($revision_1, $revision_2){
 		foreach(array_keys($revision_1->attributes) as $attribute){
 
 			// Ignore these columns
-			if(in_array($attribute, array('id', 'programme_id', 'under_review', 'status', 'created_by', 'published_by', 'created_at', 'updated_at', 'hidden', 'edits_by', 'published_at', 'made_live_by', 'instance_id'))) continue;
+			if(in_array($attribute, array('id', 'programme_id', 'under_review', 'status', 'created_by', 'published_by', 'created_at', 'updated_at', 'hidden', 'edits_by', 'published_at', 'made_live_by', 'instance_id', 'material_change', 'changelog'))) continue;
```

Comment:
stop these fields appearing in the diff PLUS stops them having some additional markup added which breaks the logic determining whether there is a material change in the view.